README.md

```diff
@@ -1,105 +1,149 @@
+| Implementation                                                                 | Requires Installation | Local Execution | HPC   | Cloud                 |
+| ------------------------------------------------------------------------------ | --------------------- | --------------- | ----- | --------------------- |
+| [AWS HealthOmics](https://docs.aws.amazon.com/omics/latest/dev/workflows.html) | Yes                   | No              | No    | AWS                   |
+| [Cromwell](https://github.com/broadinstitute/cromwell) *                        | Yes                   | Yes             | Many  | AWS Batch, Azure, GCP |
+| [dxCompiler](https://github.com/dnanexus/dxCompiler)                           | Yes                   | No              | No    | DNAnexus              |
+| [MiniWDL](https://github.com/chanzuckerberg/miniwdl)                           | Yes                   | Yes             | SLURM | AWS Batch             |
+| [Terra](https://terra.bio/)                                                    | No                    | No              | No    | Azure, GCP            |
+
+\* Also see [WDL Runner](https://github.com/broadinstitute/wdl-runner), a script for launch WDL workflows on GCP using Cromwell
+
+### Grammars, Parsers, and Language Support
+
+- The WDL [parsers repository](https://github.com/openwdl/wdl-parsers/) provides grammar definitions in various formats and generated parsers for various languages.
+- [MiniWDL](https://github.com/chanzuckerberg/miniwdl) provides python bindings for WDL as well as commandline validation and linting tools.
+- [WOMTool](https://cromwell.readthedocs.io/en/stable/WOMtool/) is a standalone Java tool for WDL parsing, validating, linting, and diagramming.
+- [wdlTools](https://github.com/dnanexus/wdlTools) - provides 1) a parser Java/Scala library, based on the  [ANTLR4 grammars](https://github.com/openwdl/wdl-parsers) grammars, for WDL draft-2, 1.0, 1.1, and 2.0; and 2) command-line tools for sytanx checking, type-checking, linting, code formatting (including upgrading from older to newer WDL versions), generating documentation, and executing WDL tasks locally.
 
 ### IDE Support
 
-- Visual Studio Code: [WDL Syntax Highlighter](https://marketplace.visualstudio.com/items?itemName=broadinstitute.wdl)
-- JetBrains IDE's: [Winstanly](https://plugins.jetbrains.com/plugin/8154-winstanley-wdl)
-- Atom: [Language-WDL](https://atom.io/packages/language-wdl)
-- Vim: [vim-wdl](https://github.com/broadinstitute/vim-wdl)
+| IDE                | Tool                                                                                             |
+| ------------------ | ------------------------------------------------------------------------------------------------ |
+| Atom               | [Language-WDL](https://atom.io/packages/language-wdl)                                            |
```

Comment:
Atom is deprecated. So I would remove this. https://github.blog/2022-06-08-sunsetting-atom/